Makefile.COMMON

```diff
@@ -37,15 +37,24 @@
 # The including Makefile may define any number of extra targets that are
 # specific to that project.
 
-VERSION ?= $(error VERSION not set in including Makefile)
```

Comment:
Oh, should've read the comment. I managed to move all the changes to the Makefile.